pages/key-manager.tsx

```diff
@@ -140,8 +143,127 @@ const KeyManager: NextPage = () => {
           </div>
         </div>
       </div>
+
+      <Nonce />
     </>
   );
 };
 
+const Nonce = () => {
```

Comment:
Can you put this component in its own separate file in the `/components/Nonce` folder? There should be a `Nonce.tsx` and an `index.ts` files